app/views/my_devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,113 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+    <% if @minimum_password_length %>
+      <br />
+      <em><%= @minimum_password_length %> characters minimum</em>
+    <% end %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :current_password %> <i>(we need your current password to confirm your changes)</i><br />
+    <%= f.password_field :current_password, autocomplete: "current-password" %>
+  </div>
+  <br />
+  <!--Personal Info from new user-->
+  <div class="col-8">
+  <div class="card text-white bg-info mb-3" style="max-width: 50rem;">
+    <div class="card-header">User Information</div>
+    <div class="card-body">  
+      <h5 class="card-title">User</h5>
+        <div>
+             <%= f.label :picture %>
+             <%= f.text_field :picture, autofocus: true %>
+        </div>
+        <div>
+             <%= f.label :first_name %>
+             <%= f.text_field :first_name, autofocus: true %>
+        
+             <%= f.label :last_name %>
+             <%= f.text_field :last_name, autofocus: true %>
+        </div>
+        <div>
+             <%= f.label :gender %>
+             <%= f.select :gender,[ 'Female', 'Male', 'Other', 'N/A' ], :prompt => 'Select One', autofocus: true %>
+        </div>   
+        <div>
+             <%= f.label :age %>
+             <%= f.select :age, options_for_select(18..100), :prompt => "Age" ,autofocus: true %>
+        </div>
+        <div>
+             <%= f.label :address %>
+             <%= f.text_field :street_address, autofocus: true %>
+        </div> 
+        <div>
+             <%= f.label :city %>
+             <%= f.text_field :city, autofocus: true %>
+
+             <%= f.label :state %>
+             <%= f.text_field :state, autofocus: true %>
+        </div>
+        <div>
+             <%= f.label :cuntry %>
```

Comment:
This is misspelled.

(Needing to correct this error in three or four different places is exactly why we encourage writing DRY ("don't repeat yourself") code -- if this was abstracted out into a single `_form` partial view, then you'd only need to correct it once.)